src/main/java/cs2103/duke/ToDo.java

```diff
@@ -0,0 +1,20 @@
+package cs2103.duke;
+
+/**
+ * This class encapsulates a Todo Task object, which inherits from the Task class.
+ */
+public class ToDo extends Task {
+    protected int index;
+    protected String description;
+
+    public ToDo(int index, String description) {
+        super(index, description);
+        this.index = index;
+        this.description = description;
```

Comment:
Similar to the previous variable, perhaps you could consider renaming `description` such that it is more self-explanatory about what it is referring to.